bin/entry.sh

```diff
@@ -4,9 +4,13 @@ if [ -z "${AWS_LAMBDA_RUNTIME_API}" ]; then
     exec /usr/bin/aws-lambda-rie $(which python) -m awslambdaric $1
 else
     . /sync_lambda_envs.sh
-    VAR_NAMES=$(env | cut -d '=' -f 1 | sort)
-    echo "ENTRY.SH: Running in AWS Lambda (. /sync_lambda_envs.sh)
-All environment variable names (from AWS Parameter Store):
-${VAR_NAMES}"
+    # Collect environment variable names (sorted) into a JSON array for a single CloudWatch structured log line
+    VAR_NAMES_NL=$(env | cut -d '=' -f 1 | sort)
+    ENV_VAR_COUNT=$(printf '%s\n' "$VAR_NAMES_NL" | grep -c '^')
+    # Build JSON array manually (avoid requiring jq at runtime)
+    VAR_NAMES_JSON=$(printf '%s\n' "$VAR_NAMES_NL" | awk 'BEGIN { printf("["); first=1 } { gsub(/"/, "\\\""); if(!first) printf(","); printf("\"%s\"", $0); first=0 } END { printf("]") }')
```

Comment:
I would also use `jo` here if possible, because it would be easier and more readable. 😛